gtda/images/filtrations.py

```diff
@@ -490,3 +490,143 @@ def transform(self, X, y=None):
         Xt = np.concatenate(Xt)
 
         return Xt
+
+
+@adapt_fit_transform_docs
+class ErosionFiltration(BaseEstimator, TransformerMixin):
+    """Filtrations of 2D/3D binary images based on the erosion of activated
+    regions.
+
+    This filtration assigns to each pixel in an image a grayscale value
+    calculated as follows. If the minimum Manhattan distance between the
+    pixel and any deactivated pixel in the image is less than or equal to
+    the parameter `n_iterations`, the assigned value is this distance –
+    in particular, deactivated pixels are assigned a value of 0.
+    Otherwise, the assigned grayscale value is the sum of the lengths
+    along all axes of the image – equivalently, it is the maximum
+    Manhattan distance between any two pixels in the image. The name of
+    this filtration comes from the fact that these values can be computed
+    by iteratively eroding activated regions, shrinking them by a total
+    amount `n_iterations`.
+
+    Parameters
+    ----------
+    n_iterations : int or None, optional, default: ``None``
+        Number of iterations in the erosion process. ``None`` means erosion
```

Comment:
Can we implement a similar rewording of this to the one made for `DilationFiltration`?

Reply:
Isn't it the same?

For DilationFIltration:
`    n_iterations : int or None, optional, default: ``None``
        Number of iterations in the dilation process. ``None`` means dilation
        over the full image.`

Reply:
I could swear we discussed a change for this in the comments...

Reply:
We did, it was ignored :) https://github.com/giotto-ai/giotto-tda/pull/289/files#r379929304

Reply:
Ahaha, sorry Umbe :)